helm/hpcc/templates/eclccserver.yaml

```diff
@@ -27,6 +27,7 @@ Pass in dict with root and me
 */}}
 {{- define "hpcc.eclccServerConfigMap" -}}
 {{- $compileJobName := printf "compile-job-_HPCC_JOBNAME_" }}
+{{- $gitPlane := .me.gitPlane | default (include "hpcc.getDefaultGitPlane" .root) -}}
```

Comment:
trivial: I notice the pre-existing line above didn't/doesn't have a whitespace chomping char , but this new line ahead of the content does.. Don't think it's problematic, perhaps make both lines same.

Reply:
I will change to match